modules/ocf/facts.d/host-env

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -euo pipefail
```

Comment:
Unfortunately, this doesn't work, because Linux treats everything after the command as one argument. So the actual command called will be:

    /bin/bash "-euo pipefail" /var/cache/puppet/facts.d/host-env

Reply:
Ah, I see. Thank you for fixing it on master!